crates/vm2/src/instruction_handlers/ret.rs

```diff
@@ -1,23 +1,34 @@
+use std::convert::Infallible;
```

Comment:
Infallible is easy to read when used in a Result but I'd use `!` or `enum Void {}` elsewhere.

Reply:
`!` is unstable and `Infallible` is [its stable replacement](https://doc.rust-lang.org/std/convert/enum.Infallible.html#future-compatibility). `enum Void {}` would technically work, but I'd rather not create additional types.